app/models/concerns/project_media_getters.rb

```diff
@@ -31,6 +31,10 @@ def is_image?
     self.is_uploaded_image?
```

Comment:
It's better to remove `is_uploaded_image?` method and move the logic inside `is_image?` method for consistent 
I know this one not related to your PR but I suggest to include this refactor.